diff --git utils/codegen/ipc/generate.py utils/codegen/ipc/generate.py
index dfbe659b..07537c4c 100755
--- utils/codegen/ipc/generate.py
+++ utils/codegen/ipc/generate.py
@@ -10,6 +10,7 @@ import os
 import sys

 sys.path.insert(0, f'{os.path.dirname(__file__)}/mojo/public/tools/bindings')
+sys.path.insert(0, os.path.dirname(__file__))

 import mojo.public.tools.bindings.mojom_bindings_generator as generator

diff --git utils/codegen/ipc/parser.py utils/codegen/ipc/parser.py
index 8e70322d..4c0da572 100755
--- utils/codegen/ipc/parser.py
+++ utils/codegen/ipc/parser.py
@@ -11,6 +11,7 @@ import sys

 # Make sure that mojom_parser.py can import mojom
 sys.path.insert(0, f'{os.path.dirname(__file__)}/mojo/public/tools/mojom')
+sys.path.insert(0, os.path.dirname(__file__))

 import mojo.public.tools.mojom.mojom_parser as parser
 